plugins/outputs/sensu/README.md

```diff
@@ -0,0 +1,85 @@
+# HTTP Output Plugin
+
+This plugin writes metrics events to [Sensu Go](https://sensu.io)  via its 
+HTTP events API.
+
+### Configuration:
+
+```toml
+  ## BACKEND API URL is the Sensu Backend API root URL to send metrics to 
+  ## (protocol, host, and port only). The output plugin will automatically 
+  ## append the corresponding backend or agent API path (e.g. /events or 
+  ## /api/core/v2/namespaces/:entity_namespace/events/:entity_name/:check_name).
+  ## 
+  ## NOTE: if backend_api_url and agent_api_url and api_key are set, the output 
+  ## plugin will use backend_api_url. If backend_api_url and agent_api_url are 
+  ## not provided, the output plugin will default to use an agent_api_url of 
+  ## http://127.0.0.1:3031
+  ## 
+  # backend_api_url = "http://127.0.0.1:8080"
+  # agent_api_url = "http://127.0.0.1:3031"
```

Comment:
Could you also add a sentence regarding `agent_api_url`. Sorry I don't know the details of Sensu (yet) but I don't understand how `backend_api_url` and `agent_api_url` differ and what to use when...